src/__test__/Toast/Toast.test.js

```diff
@@ -17,7 +17,7 @@ describe('Toasts', () => {
   })
 
   it('Should pass additional classNames', () => {
-    const wrapper = shallow(<Toast className='extra'>My Toasts</Toast>)
+    const wrapper = shallow(<Toast className="extra">My Toasts</Toast>)
```

Comment:
Inconsistent quotes here. Use single quotes, follow the StandardJS style, run `npm run lint` to check for errors or use the extension.